projects/packages/jetpack-mu-wpcom/src/class-jetpack-mu-wpcom.php

```diff
@@ -99,6 +99,7 @@ public static function load_features() {
 			require_once __DIR__ . '/features/help-center/class-help-center.php';
 		}
 		require_once __DIR__ . '/features/import-customizations/import-customizations.php';
+		require_once __DIR__ . '/features/jetpack-global-styles/class-global-styles.php';
```

Comment:
We decided to load features from ETK via `load_etk_features` for now to ensure those features are loaded earlier before loading ETK plugin. See https://github.com/Automattic/jetpack/pull/38230.